integration-tests/elytron-security-ldap/src/test/java/io/quarkus/elytron/security/ldap/it/ElytronSecurityLdapTest.java

```diff
@@ -82,4 +87,26 @@ void admin_role_not_authorized() {
                 .then()
                 .statusCode(403);
     }
+
+    @Test()
+    void standard_role_authenticated_cached() throws LDAPException {
+        RestAssured.given()
+                .redirects().follow(false)
+                .when()
+                .auth().preemptive().basic("standardUser", "standardUserPassword")
+                .get("/api/requiresStandardRole")
+                .then()
+                .statusCode(200);
+
+        ldapServer.shutDown(false);
```

Comment:
this uses default test method order, so if you stop this server here, other test method results can be corrupted (yet succeed?). wdyt can you make this run as the last, please?

Reply:
You are right. It is pure luck that this test runs last.

So i can either user the `@Order` annotation on all the tests in the class or simply start the server again in the end of the test.

@michalvavrik do you have any preferences?

Reply:
both is fine with me as it is in memory handler, it should be quick. personally I'd go with `@Order` but you are the PR author!

Reply:
Is done.